shopify/actions/actions_schema.graphql

```diff
@@ -4,16 +4,35 @@ type ProductOption {
   values: [String]
 }
 
+type Money {
+  ## Decimal money amount.
+  amount: Currency
+  ## Currency of the money.
+  currencyCode: String
+}
+
+## A collection of monetary values in their respective currencies. 
+## Typically used in the context of multi-currency pricing and transactions, 
+## when an amount in the shop's currency is converted to the customer's currency of choice.
+type MoneySet {
```

Comment:
How does this value work? The description says it's a collection of values but the shape of the object looks like just one value

Reply:
I guess this is just the shape that Shopify does. Perhaps there is other fields in here (e.g. customer's currency value) that would make this a set that were not included?

Reply:
It looks like there is shopMoney and presentmentMoney. I think both of these are important as one is how much customer paid in their currency and the other one is how much they paid in shop's currency